docs/api/browser-window.md

```diff
@@ -983,6 +983,29 @@ Same as `webContents.capturePage([rect, ]callback)`.
   * `httpReferrer` String - A HTTP Referrer url.
   * `userAgent` String - A user agent originating the request.
   * `extraHeaders` String - Extra headers separated by "\n"
+  * `postData` Array - An array of `upload` objects which
```

Comment:
@MarshallOfSound this notation seems good to me. My only concern is that we're putting more pressure on docs authors to be intimately aware of the documentation styleguide, but I guess that's necessary if we want to be able to extract structured data from all the docs.

Maybe we could use this new conditional notation, but also leave in the prose in an abbreviated form for the sake of clarity, like this:

```
If `type` is `data`, then `upload` must be an [UploadData](structures/upload-data.md) structure.
etc..
```